eng/scripts/Update-Snippets.ps1

```diff
@@ -19,8 +18,20 @@ if ($ServiceDirectory -and ($ServiceDirectory -ne "*")) {
 
 if (-not (Test-Path env:TF_BUILD)) { $StrictMode = $true }
 
-if($StrictMode) {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -sm -c Release }
-} else {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -c Release }
+nuget install SnippetGenerator -Source azure-sdk-tools@DevOps -PreRelease -OutputDirectory .
```

Comment:
Do we have plans to use this script outside of the tools repo? If so we need to not have a dependency on a feed name but instead we probably need to give the feed path here.